src/main/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipService.java

```diff
@@ -243,28 +245,30 @@ public void addNotifications(ProgrammeMembership programmeMembership)
   private void createDirectProgrammeNotifications(ProgrammeMembership programmeMembership,
       Map<NotificationType, History> notificationsAlreadySent) throws SchedulerException {
 
-    //only handle 'programme created' notifications
-    boolean shouldSchedule = shouldScheduleNotification(PROGRAMME_CREATED, programmeMembership,
-        notificationsAlreadySent);
+    LocalDate startDate = programmeMembership.getStartDate();
+
+    JobDataMap jobDataMap = new JobDataMap();
+    jobDataMap.put(TIS_ID_FIELD, programmeMembership.getTisId());
+    jobDataMap.put(PERSON_ID_FIELD, programmeMembership.getPersonId());
+    jobDataMap.put(PROGRAMME_NAME_FIELD, programmeMembership.getProgrammeName());
+    jobDataMap.put(PROGRAMME_NUMBER_FIELD, programmeMembership.getProgrammeNumber());
+    jobDataMap.put(START_DATE_FIELD, startDate);
+    jobDataMap.put(TEMPLATE_OWNER_FIELD, programmeMembership.getManagingDeanery());
+    if (programmeMembership.getConditionsOfJoining() != null) {
+      jobDataMap.put(COJ_SYNCED_FIELD,
+          programmeMembership.getConditionsOfJoining().syncedAt());
+    }
+    // Note the status of the trainee will be retrieved when the job is executed, as will
+    // their name and email address and LO contact details.
 
-    if (shouldSchedule) {
+    // PROGRAMME_CREATED
+    boolean shouldScheduleProgrammeCreated = shouldScheduleNotification(PROGRAMME_CREATED,
+        programmeMembership, notificationsAlreadySent);
+    if (shouldScheduleProgrammeCreated) {
       log.info("Processing notification {} for {}.", PROGRAMME_CREATED,
           programmeMembership.getTisId());
 
-      JobDataMap jobDataMap = new JobDataMap();
-      jobDataMap.put(TIS_ID_FIELD, programmeMembership.getTisId());
-      jobDataMap.put(PERSON_ID_FIELD, programmeMembership.getPersonId());
-      jobDataMap.put(PROGRAMME_NAME_FIELD, programmeMembership.getProgrammeName());
-      jobDataMap.put(PROGRAMME_NUMBER_FIELD, programmeMembership.getProgrammeNumber());
-      jobDataMap.put(START_DATE_FIELD, programmeMembership.getStartDate());
-      jobDataMap.put(TEMPLATE_OWNER_FIELD, programmeMembership.getManagingDeanery());
       jobDataMap.put(TEMPLATE_NOTIFICATION_TYPE_FIELD, PROGRAMME_CREATED);
```

Comment:
Create a new map from the existing `jobDataMap` map of shared values, otherwise in an edge case where a PM is created on day one we'll include extra variables in the day one notification.